### PR TITLE
Added externs for the Stativus statechart library.

### PR DESCRIPTION
**Pull request by OliverM**
New package:

**Extern:** Generated.

### CODE REVIEW

File: stativus/build.boot [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ pom  {:project     'cljsjs/stativus
+       :version     +version+
+       :description "A Statechart library"
+       :url         "https://github.com/etgryphon/stativus"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"BSD" "http://opensource.org/licenses/BSD-3-Clause"}})
+
+(deftask package []
+  (comp
+   (download :url "https://github.com/etgryphon/stativus/archive/master.zip"
```
Comment (Deraen) [inline stativus/build.boot]: Please use `+lib-version+` when downloading the archive, e.g. https://github.com/etgryphon/stativus/archive/0.10.0.zip

File: stativus/build.boot [hunk 2]
```diff
@@ -0,0 +1,29 @@
+(set-env!
+ :resource-paths #{"resources"}
+ :dependencies '[[cljsjs/boot-cljsjs "0.5.2"  :scope "test"]])
+
+(require '[cljsjs.boot-cljsjs.packaging :refer :all])
+
+(def +lib-version+ "0.1.0")
```
Comment (Deraen) [inline stativus/build.boot]: This should match the library version, currently 0.10.0.

**Comment by OliverM** [comment]
Thanks Derean, didn't realise you could pull down a version-specific zip via github like that...

File: stativus/build.boot [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ pom  {:project     'cljsjs/stativus
+       :version     +version+
+       :description "A Statechart library"
+       :url         "https://github.com/etgryphon/stativus"
+       :scm         {:url "https://github.com/cljsjs/packages"}
+       :license     {"BSD" "http://opensource.org/licenses/BSD-3-Clause"}})
+
+(deftask package []
+  (comp
+   (download :url "https://github.com/etgryphon/stativus/archive/0.10.0.zip"
```
Comment (Deraen) [inline stativus/build.boot]: Please use `format` (or `str`) to build the URL here using `+lib-version+`, `(format "https://github.com/etgryphon/stativus/archive/%s.zip" +lib-version+)`, this way only the var needs be updated when updating the package and old archive won't get used accidentally.

**Pull request merged**